tests/strategies/test_generate_kb.py

```diff
@@ -97,7 +94,8 @@ def test_generate_kb():
             "kb_document_class": ":MyData",
             "kb_document_update": {"dataresource": {"license": "MIT"}},
             "kb_document_base_iri": "http://ex.com#",
-            "kb_document_context": {EMMO.isDescriptionFor: ":MyMaterial"},
+            # This cannot be found in EMMO, so it will be ignored...
+            # "kb_document_context": {EMMO.isDescriptionFor: ":MyMaterial"},
```

Comment:
It exists in the development version (release candidate of EMMO, so best to keep it here).

Reply:
So, after a bit more investigation of the code, I disagree with my comment above. 
Since we have set check=True in the namespace, it has  to be commented out for now. 
I make a small suggestion for changes in the comment.